lib/button/resources/resource.rb

```diff
@@ -43,10 +44,13 @@ def timeout
     # Performs an HTTP GET at the provided path.
     #
     # @param [String] path the HTTP path
+    # @param [Hash=] query optional query params to send
     # @return [Button::Response] the API response
     #
-    def api_get(path)
-      api_request(Net::HTTP::Get.new(path))
+    def api_get(path, query = nil)
+      uri = URI(path)
+      uri.query = URI.encode_www_form(query) if query
+      api_request(Net::HTTP::Get.new(uri.to_s))
```

Comment:
I do, as far as I can tell.  Passing a `URI` causes a `Cannot duplicate: NilClass` error (I didn't check across MRI versions).  I dug around in rubydocs for awhile and didn't find anything (and in fact found snippets indicating passing `URI` is okay) describing the interface sufficiently that I could work things out.  I'll take another look today just to be sure.